File: client/models/AnnouncementReply.ts
export interface AnnouncementReply {
  replyId: number;
  announcementId: number;
  membershipId: number;
  message: string;
  repliedAt: string;
  memberName: string;
}

export interface AnnouncementReplyCreateRequest {
  announcementId: number;
  message: string;
}

File: api/src/models/announcement_reply.py
from pydantic import BaseModel
from datetime import datetime

class AnnouncementReplyCreateRequest(BaseModel):
    announcement_id: int
    message: str

class AnnouncementReply(BaseModel):
    reply_id: int
    announcement_id: int
    membership_id: int
    message: str
    replied_at: datetime

class AnnouncementReplyResponse(BaseModel):
    reply_id: int
    announcement_id: int
    membership_id: int
    message: str
    replied_at: datetime
    member_name: str
